app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = current_user
+    @posts = current_user.last_three_posts
+  end
 
-  def show; end
+  def show
+    @post = Post.find params[:id]
+    @comments = @post.recent_comments
+  end
```

Comment:
- [ ] The post show action should get all the comments for a post as seen in the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png). Kindly fix this